tracer_scripts/query_execution_time.d

```diff
@@ -0,0 +1,13 @@
+#!/usr/sbin/dtrace -s 
```

Comment:
How do I run this?
```
[:~/Projekte/Opossum/Git] mp18/usdt+ ± sudo ./tracer_scripts/query_execution_time.d -c ./build/hyriseConsole
dtrace: system integrity protection is on, some features will not be available

[PERF] Hyrise is running as a debug build. at src/lib/utils/performance_warning.cpp:9
	Performance can be affected. This warning is only shown once.

dtrace: failed to compile script ./tracer_scripts/query_execution_time.d: line 3: probe description hyrise34815:::query_start does not match any probes
[:~/Projekte/Opossum/Git] mp18/usdt+ ± grep -r query_start src
src/lib/utils/tracing/provider.d:        probe query_start(char* query, int task_size, uintptr_t this_pointer);
```

Reply:
System integrity protection is enabled on your device. This prohibits the usage of USDT with dtrace. I added a paragraph to the [wiki](https://github.com/hyrise/hyrise/wiki/Profiling#mac-os). Look at this [post](https://stackoverflow.com/a/33584192) to solve this issue.